app/utils.py

```diff
@@ -37,13 +37,11 @@ def default(self, obj):
         return encoder(obj)
 
 
-def oxford_join(items):
+def oxford_join(items: List):
     "joins a list with a comma, but uses 'and' before the last item"
-    if (len(items) < 1):
-        return None
+    if not isinstance(items,list) or len(items) < 1:

```

Comment:
Hmm, I'm not sure about best practices for runtime type checking when static type checking is in use. This would probably be a question for a team to answer together, and I think I'd encourage dropping runtime checks in this case.

You might want to consider widening this type to `typing.Sequence` based on a quick glance at this table: https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes

Reply:
Point taken. There's a longer story here which I'll hit you with soon. I think I should use error handling instead, but I was over valuing brevity...

Reply:
ok, here's the story.

essentially, i was going for maximal simplicity/brevity in the following helper function:

```python
def get_answers(template_index,right_game_id):
    template = question_templates[template_index]
    right_game = all_games_by_id[right_game_id]
    right_answer = getattr(right_game, template["answer_type"])
    answers = set()
    answers.add(oxford_join(right_answer)) # ox
    while len(answers) < 4:
        if template["answer_type"] == "year_published":
            # is the complication of code worth the extent to which this is 
            # better than sampling random games?
            alt_answer = random.randint(
                right_game.year_published-10, 
                min(datetime.now().year, right_game.year_published+10)
            )
        else:
            alt_answer = getattr(
                random.choice(all_games), 
                template["answer_type"]
            )
        answers.add(oxford_join(alt_answer)) # ox
    return list(answers)
```

I'm calling `oxford_join(right_answer/alt_answer)` because in the case where ` template["answer_type"] == "designer"`, right_answer/alt_answer is a list. For some reason I decided it was neater to have `oxford_join` take any param type and return an appropriate value, rather than limiting the calls to it.

to remedy, would you prefer `try/catch` where oxford_join throws a `TypeError` if it gets an int or string, or something like `right_answer/alt_answer = oxford_join(right_answer/alt_answer) if template["answer_type"] == "designer" else right_answer/alt_answer`

Reply:
Thinking through the various options – I feel like the cleanest next thing might be to make every answer a list, where a list of a single item is treated the same way a string used to be, and there are no special cases.

Reply:
i ended up doing 

```python
if template["answer_type"] == "developers":
            right_answer = oxford_join(right_answer) 
        answers.add(right_answer)
```

for the time being